### PR TITLE
[16.0][MIG] stock_quant_package_dimension_total_weight_from_packaging: Migrate to 16.0

### PR DESCRIPTION
**Pull request by QuocDuong1306**
Depend on:

- https://github.com/OCA/product-attribute/pull/1404

### CODE REVIEW

**Comment by rousseldenis** [comment]
/ocabot migration stock_quant_package_dimension_total_weight_from_packaging

File: setup/stock_quant_package_dimension_total_weight_from_packaging/odoo/addons/stock_quant_package_dimension_total_weight_from_packaging [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../stock_quant_package_dimension_total_weight_from_packaging
```
Comment (rousseldenis) [inline setup/stock_quant_package_dimension_total_weight_from_packaging/odoo/addons/stock_quant_package_dimension_total_weight_from_packaging:1]: For the next migration, these changes should go into a separate commit.

See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch

Comment (QuocDuong1306) [reply setup/stock_quant_package_dimension_total_weight_from_packaging/odoo/addons/stock_quant_package_dimension_total_weight_from_packaging:1]: Thank @rousseldenis , i will take notes for the next time

Comment (QuocDuong1306) [reply setup/stock_quant_package_dimension_total_weight_from_packaging/odoo/addons/stock_quant_package_dimension_total_weight_from_packaging:1]: Thank @rousseldenis , I updated

**Review by cyrilmanuel — APPROVED** [review]

**Comment by cyrilmanuel** [comment]
@QuocDuong1306  hello could you do a small change on that module and save it ? it shall fix the tests error

**Review by cyrilmanuel — APPROVED** [review]

**Review by santostelmo — APPROVED** [review]

**Review by Ricardoalso — APPROVED** [review]

**Review by cyrilmanuel — APPROVED** [review]
